internal/service/external_service_rule_test.go

```diff
@@ -517,10 +519,10 @@ func TestRestService(t *testing.T) {
 	}, 0)
 }
 
-func jsonOut(w http.ResponseWriter, err error, out interface{}) {
+func jsonOut(w http.ResponseWriter, _ error, out interface{}) {
```

Comment:
just remove the error parameter if useless

Reply:
done.